### PR TITLE
Fix/remove unused props and attributes

### PR DESCRIPTION
**Pull request by dopenguin**
## Summary

Removed unused or redundant props, removed orphaned configuration parameters and adjusted documentation and types to properly reflect optionality while also adding all configurable parameters to the documentation. 
For more information, take a look at the respective changelogs.

## Instructions for local reproduction and review

DISH, Meldemichel and snowbox have been updated. All changes are non-breaking, thus the behaviour shouldn't have changed.

### CODE REVIEW

**Review by warm-coolguy — CHANGES_REQUESTED** [review]
🏓 @dopenguin

File: packages/clients/dish/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## unpublished
 
 - Feature: If a user is geolocated outside of the map's extent, the client will inform the user of why geolocation did not take effect via a textbox.
+- Refactor: Adjust dish-gfi-component to the new API.
```
[the remark this answers is not in the document]
Comment (dopenguin) [reply packages/clients/dish/CHANGELOG.md]: Agreed 57dae6e73f7faef4896361a35fdd274c467ac168

File: packages/clients/meldemichel/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## unpublished
+
+- Fix: Adjust types to correctly reflect usages.
+- Refactor: Adjust meldemichel-gfi-component to the new API.
```
[the remark this answers is not in the document]
Comment (dopenguin) [reply packages/clients/meldemichel/CHANGELOG.md]: Agreed 57dae6e73f7faef4896361a35fdd274c467ac168

File: packages/plugins/Gfi/README.md [hunk 3]
```diff
@@ -4,62 +4,56 @@
 
 The GFI plugin can be used to fetch and optionally display GFI (GetFeatureInfo) from WMS and WFS services as well as layers based on GeoJSON files. In a first step, the information is stored in the VueX store to allow for easy access. Display is optional and quite obstructive within the map client.
 
-> ⚠️ The display feature is currently not meant for production. Please use data bindings for display to avoid obstructing the map client.
```
[the remark this answers is not in the document]
Comment (dopenguin) [reply packages/plugins/Gfi/README.md:7]: Added that to the agenda

File: packages/lib/getFeatures/types.ts [hunk 4]
```diff
@@ -41,25 +42,23 @@ export interface SearchParameters {
...
 
 /** Parameters for wfs searches */
-export interface WfsParameters {
+export interface WfsParameters extends QueryParameters {
+  /** Feature prefix from xmlns namespace without :; e.g. 'ave' */
+  featurePrefix: string
   /** Name of the type of features (see ?service=wfs&request=DescribeFeatureType) */
   typeName: string
+  /** Namespace of feature type */
+  xmlns: string
```
[the remark this answers is not in the document]
Comment (dopenguin) [reply packages/lib/getFeatures/types.ts:51]: It is first sorted by required and optional parameters, then alphabetically.

**Comment by dopenguin** [comment]
> 🏓 @dopenguin

🏓 @warm-coolguy 

We can further work on this PR after the next weekly.

File: packages/plugins/Gfi/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (dopenguin) [reply packages/plugins/Gfi/README.md]: Re-added the information cb5febcbe74f1c82a3002c459da9cd60c891f9a6

**Comment by dopenguin** [comment]
> > 🏓 @dopenguin
> 
> 🏓 @warm-coolguy
> 
> We can further work on this PR after the next weekly.

🏓 @warm-coolguy

**Review by warm-coolguy — APPROVED** [review]
> > > 🏓 @dopenguin
> > 
> > 
> > 🏓 @warm-coolguy
> > We can further work on this PR after the next weekly.
> 
> 🏓 @warm-coolguy

🚀 @dopenguin

**Pull request merged**